Tests/LibWeb/Text/expected/wpt-import/html/infrastructure/safe-passing-of-structured-data/window-postmessage.window.txt

```diff
@@ -137,7 +137,7 @@ Pass	ObjectPrototype must lose its exotic-ness when cloned
 Pass	Serializing a non-serializable platform object fails
 Pass	An object whose interface is deleted from the global must still deserialize
 Pass	A subclass instance will deserialize as its closest serializable superclass
-Fail	Resizable ArrayBuffer
+Pass	Resizable ArrayBuffer
 Fail	Growable SharedArrayBuffer
```

Comment:
wait, didn't you add growable shared array buffer here? why doesn't this one pass? 🤔

Reply:
cross origin isolation :) Please, see my comment [here](https://github.com/LadybirdBrowser/ladybird/pull/2768#issuecomment-2518536986).

Reply:
Also, in `LibWeb/Text/input/wpt-import/common/sab.js` the following code exists:
```js
onst createBuffer = (() => {
  // See https://github.com/whatwg/html/issues/5380 for why not `new SharedArrayBuffer()`
  let sabConstructor;
  try {
    sabConstructor = new WebAssembly.Memory({ shared:true, initial:0, maximum:0 }).buffer.constructor;
  } catch(e) {
    sabConstructor = null;
  }
  return (type, length, opts) => {
    if (type === "ArrayBuffer") {
      return new ArrayBuffer(length, opts);
    } else if (type === "SharedArrayBuffer") {
      if (sabConstructor && sabConstructor.name !== "SharedArrayBuffer") {
        throw new Error("WebAssembly.Memory does not support shared:true");
      }
      return new sabConstructor(length, opts);
    } else {
      throw new Error("type has to be ArrayBuffer or SharedArrayBuffer");
    }
  }
})();
```

Reply:
@ADKaster [these](https://wpt.fyi/results/wasm/jsapi/memory/constructor-shared.tentative.any.html?label=master&product=ladybird) `WebAssembly.Memory` tests seem to be related.

Reply:
I will try to implement shared memory allocation with `WebAssembly` over the weekend. My only concern is:
> The shared attribute is only documented in [the Threading proposal for WebAssembly](https://github.com/WebAssembly/threads/blob/master/proposals/threads/Overview.md#javascript-api-changes) and not part of the official specs.
see [this](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/Memory/Memory#specifications)